diff --git a/MSCL/source/mscl/Endianness.h b/MSCL/source/mscl/Endianness.h
index 5bc5027c0..626127626 100644
--- a/MSCL/source/mscl/Endianness.h
+++ b/MSCL/source/mscl/Endianness.h
@@ -42,9 +42,23 @@ MIT Licensed. See the included LICENSE.txt for a copy of the full MIT License.
 
 #pragma once
 
-#include <boost/detail/endian.hpp>
 #include <algorithm>
 
+#include <boost/predef/other/endian.h>
+
+#if BOOST_ENDIAN_BIG_BYTE
+#   define BOOST_BIG_ENDIAN
+#   define BOOST_BYTE_ORDER 4321
+#endif
+#if BOOST_ENDIAN_LITTLE_BYTE
+#   define BOOST_LITTLE_ENDIAN
+#   define BOOST_BYTE_ORDER 1234
+#endif
+#if BOOST_ENDIAN_LITTLE_WORD
+#   define BOOST_PDP_ENDIAN
+#   define BOOST_BYTE_ORDER 2134
+#endif
+
 namespace mscl
 {
 
diff --git a/MSCL/source/stdafx.h b/MSCL/source/stdafx.h
index 139ba571f..10fc10864 100644
--- a/MSCL/source/stdafx.h
+++ b/MSCL/source/stdafx.h
@@ -15,7 +15,7 @@ MIT Licensed. See the included LICENSE.txt for a copy of the full MIT License.
 #include <boost/date_time.hpp>                                     //for boost::posix_time::from_time_t
 #include <boost/date_time/posix_time/posix_time_duration.hpp>      //for boost::posix_time::nanosec
 #include <boost/date_time/posix_time/ptime.hpp>                    //for boost ptime    
-#include <boost/detail/endian.hpp>                                 //for endianess
+#include <boost/predef/other/endian.h>                             //for endianess
 #include <boost/numeric/conversion/cast.hpp>                       //for boost::numeric_cast
 #include <boost/optional.hpp>                                      //for boost::optional
 #include <boost/utility/binary.hpp>                                //for BOOST_BINARY
